.github/workflows/artifact-cleanup.yml

```diff
@@ -1,15 +0,0 @@
-# WARNING: This file is autogenerated - changes will be overwritten if not made via https://github.com/pulumi/ci-mgmt
-
-name: cleanup
-on:
-  schedule:
-  - cron: 0 1 * * *
-jobs:
-  remove-old-artifacts:
-    runs-on: ubuntu-latest
-    steps:
-    - name: Remove old artifacts
-      uses: c-hive/gha-remove-artifacts@v1
-      with:
-        age: 1 month
-        skip-tags: true
```

Comment:
We can remove this workflow because we use the expiry in the upload-artifact action instead ([Refactor artifact cleanup · ci-mgmt/484](https://github.com/pulumi/ci-mgmt/pull/484))